components/baseforms/FInput.vue

```diff
@@ -53,6 +63,12 @@ export default {
       this.icon && (optProps.icon = this.icon)
       return optProps
     },
+    fieldProps() {
+      const optProps = {}
+      this.fieldType && (optProps.type = this.fieldType)
+      this.fieldMessage && (optProps.message = this.fieldMessage)
+      return optProps
```

Comment:
I wonder if it might be more straightforward to not have a default on these props and then just make `fieldProps` the values passed (which would be undefined if not set)? Or does the presence of those keys mess with something even if undefined?